### PR TITLE
Protobufs and Telemetry Tasks

### PR DESCRIPTION
**Pull request by shivamdesai04**

### CODE REVIEW

**Comment by shivamdesai04** [comment]
Still have to poll sensors, pressure transducer and thermocouple. Tasks are not on this branch so cant yet add it

**Comment by shivamdesai04** [comment]
Should i merge in the sensors stuff here or wait for it to be merged into main and then add it here

File: Components/Communication/Inc/UARTTask.hpp [hunk 1]
```diff
@@ -16,7 +16,8 @@
 enum UART_TASK_COMMANDS {
 	UART_TASK_COMMAND_NONE = 0,
 	UART_TASK_COMMAND_SEND_DEBUG,
-	UART_TASK_COMMAND_MAX
+	UART_TASK_COMMAND_MAX,
+	UART_TASK_COMMAND_SEND_RADIO
```
Comment (cjchanx) [inline Components/Communication/Inc/UARTTask.hpp:20]: Fine for now, maybe change to SEND_PROTO later

File: Components/Core/Command.cpp [hunk 2]
```diff
@@ -83,9 +83,9 @@ Command::Command(GLOBAL_COMMANDS command, uint16_t taskCommand)
 /**
```
Comment (cjchanx) [inline Components/Core/Command.cpp:83]: Update with latest

File: Components/FlightControl/FlightTask.cpp [hunk 3]
```diff
@@ -66,6 +67,21 @@ void FlightTask::Run(void * pvParams)
         GPIO::LED1::Off();
         osDelay(500);
 
+        // For testing, generate a PROTOBUF message and send it to the Protocol Task
+		Proto::ControlMessage msg;
+		msg.set_source(Proto::Node::NODE_DMB);
```
Comment (cjchanx) [inline Components/FlightControl/FlightTask.cpp]: Source is wrong

**Review by cjchanx — APPROVED** [review]

**Pull request merged**